Dockerfile.development

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.11-stretch
+
+ENV _RUBY_VERSION 2.5.3
+ENV _RBENV_VERSION  v1.1.1
+ENV _RBUILD_VERSION v20190130
+
+# Ruby and mruby deps
+RUN apt update && \
+    apt -y upgrade && \
+    apt -y install bzip2 autoconf bison build-essential libssl-dev libyaml-dev \
+                   libreadline-dev zlib1g-dev libncurses5-dev libffi-dev \
+                   libgdbm3 libgdbm-dev
+
+RUN git clone https://github.com/sstephenson/rbenv.git --branch $_RBENV_VERSION ~/.rbenv
+RUN cd ~/.rbenv && src/configure && make -C src
+ENV PATH ~/.rbenv/bin:~/.rbenv/shims:$PATH
+RUN echo 'eval "$(rbenv init -)"' >> /etc/profile.d/rbenv.sh # or /etc/profile
+RUN echo 'eval "$(rbenv init -)"' >>  ~/.bashrc
+
+RUN git clone https://github.com/sstephenson/ruby-build.git --branch $_RBUILD_VERSION ~/.rbenv/plugins/ruby-build
+RUN ~/.rbenv/plugins/ruby-build/install.sh
+RUN eval ~/.rbenv/bin/rbenv install $_RUBY_VERSION && ~/.rbenv/bin/rbenv global $_RUBY_VERSION
+
+ENV APP ${GOPATH}/src/github.com/anycable/anycable-go
+COPY . ${APP}
+WORKDIR ${APP}
+
+RUN eval "$(~/.rbenv/bin/rbenv init -)" && make prepare
```

Comment:
Omg, `rbenv` 🙀

Why do we need Ruby for? Building mruby stuff? Then default system Ruby (`apt-get install ruby`) should be enough (I had the following Dockerfile for mruby project: https://github.com/palkan/acli/blob/master/Dockerfile).

Reply:
Debian is too conservative with packages. The current version is 2.3 https://packages.debian.org/stretch/ruby. It’s not compatible with anyt.

Reply:
Busted!